doc/version_history.rst

```diff
@@ -6,6 +6,23 @@
 Version History
 ###############
 
+v0.16.0
+-------
+
+Changes:
+
+* Update for new telemetry data from the low-level controller.
+  This version requires ts_remote_controller v1.2.2.
```

Comment:
Thanks for the fix.

I prefer to leave it out of "requires" because I intend the requirements section to be about importing software libraries. But it's a good idea to figure out a way to document this.